README.md

```diff
@@ -90,6 +91,27 @@ def show_unit(juju_show_units):
 
 **Remember**: `juju-doctor` will only run functions that exactly match a supported artifact name, and will always pass to them a dictionary of *model name* mapped to the proper artifact.
 
+### Ruleset
+Ruleset probes are written in YAML, specifying which probes should be coordinated for a deployment validation.
+
+Currently, we support the following probe types:
+- **`scriptlet`**: A Python probe
+- **`ruleset`**: A declarative deployment RuleSet
+- **`directory`**: A directory of probes (from the types in this list)
+
+```yaml
+name: A declarative deployment RuleSet
+probes:
+  - name: Local probe - passing
+    type: scriptlet
+    uri: file://tests/resources/probes/python/passing.py
```

Comment:
Per [wikipedia](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Example_URIs), strictly speaking, these are URLs, not URIs.